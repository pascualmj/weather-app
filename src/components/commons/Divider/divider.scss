@import "../../../assets/scss/variables";

.divider {
  height: 1px;
  background-color: $color-divider;
}
