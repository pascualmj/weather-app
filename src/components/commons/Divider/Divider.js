import React from "react";
import "./divider.scss";

const Divider = () => <div className="divider"></div>;

export default Divider;
