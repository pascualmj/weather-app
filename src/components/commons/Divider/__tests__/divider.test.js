import React from "react";
import { render } from "@testing-library/react";
import "@testing-library/jest-dom";
import Divider from "../Divider";

describe("Component: Divider", () => {
  test("Should match snapshot.", () => {
    const { asFragment } = render(<Divider />);

    expect(asFragment()).toMatchSnapshot();
  });
});
