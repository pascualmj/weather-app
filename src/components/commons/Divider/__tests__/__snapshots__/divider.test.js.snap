// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Divider Should match snapshot. 1`] = `
<DocumentFragment>
  <div
    class="divider"
  />
</DocumentFragment>
`;
